app/assets/stylesheets/common/base/topic.scss

```diff
@@ -110,15 +110,14 @@
 
 .progress-back-container {
   z-index: z("dropdown");
-  margin-right: 0;
   animation-duration: 0.5s;
   animation-name: button-jump-up;
-  width: 145px;
```

Comment:
Dropping this and anchoring to the right gives more consistent results. The width of the topic progress element relies on number of topics (102 / 155 is larger than 1 / 10, for example) and on whether user is admin or not.